automation/unit-tests.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -ex
 
+# dependency check
+pip show openshift
```

Comment:
Can you make it quiet, so if openshift is installed, it would not print package information into the log?
```bash
pip show -q openshift
```

Reply:
Done